mobileapp/www/js/index.js

```diff
@@ -40,12 +46,16 @@ function onDeviceReady() {
   } catch (ex) {
     console.log('Lock orientation not supported');
   }
+
 }
 
 var app = {
   // Application Constructor
   init: function init() {
     console.log('app initializing!: ', arguments);
+    if (window.localStorage.touchidLogin == 1){
```

Comment:
This is fine, but be aware that all values in localStorage are strings - this works via coercion by ==

Also, use "touchIdLoginEnabled"